src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheck.java

```diff
@@ -708,22 +698,6 @@ private static boolean isVariableInOperatorExpr(
             ast = ast.getNextSibling();
         }
 
-        // Variable may be met in ELSE declaration
-        // So, check variable usage in these declarations.
-        if (!isVarInOperatorDeclaration) {
-            final DetailAST elseBlock = operator.getLastChild();
-
-            if (elseBlock.getType() == TokenTypes.LITERAL_ELSE) {
-                // Get IF followed by ELSE
-                final DetailAST firstNodeInsideElseBlock = elseBlock.getFirstChild();
-
-                if (firstNodeInsideElseBlock.getType() == TokenTypes.LITERAL_IF) {
-                    isVarInOperatorDeclaration =
-                        isVariableInOperatorExpr(firstNodeInsideElseBlock, variable);
-                }
```

Comment:
these were surviving mutation after my changes as this part of the function was useful when it was used by `getDistToVariableUsageInChildNode`  but after debugging it was redundant when used by `calculateDistanceBetweenScopes` as the work done here to recurse down to check every node (else statments) in If-else block. can be covered  by the method `calculateDistanceBetweenScopes`. as there is a loop in it that checks  If variable usage exists in a single scope, then look into this scope and update the current scope in each iteration so we checking all the else statements in it

 this mutation didn't survive before becuase this recursion with neccessary for `getDistToVariableUsageInChildNode`  because we we don't have such a loop in this method that checks every node (else statements) we were just passing the first If node and let the recursion do the work for us